shopinvader_restapi/models/shopinvader_backend.py

```diff
@@ -70,28 +64,6 @@ class ShopinvaderBackend(models.Model):
         "field on the sale order created.",
     )
     filter_ids = fields.Many2many(comodel_name="product.filter", string="Filter")
```

Comment:
`product.filter` is now defined in `shopinvader_product_binding`.

Reply:
thanks for the feedback, [53839ec](https://github.com/shopinvader/odoo-shopinvader/pull/1382/commits/53839ec0e3327c407058187016422a6e8f7dfec1)